Mathlib/Order/Filter/FilterProduct.lean

```diff
@@ -37,27 +37,29 @@ open Ultrafilter
 
 local notation "β*" => Germ (φ : Filter α) β
 
-instance groupWithZero [GroupWithZero β] : GroupWithZero β* :=
-  { Germ.divInvMonoid, Germ.monoidWithZero with
-    mul_inv_cancel := fun f => inductionOn f fun f hf => coe_eq.2 <|
-      (φ.em fun y => f y = 0).elim (fun H => (hf <| coe_eq.2 H).elim) fun H =>
-        H.mono fun x => mul_inv_cancel
-    inv_zero := coe_eq.2 <| by
-       simp only [Function.comp, inv_zero]
-       exact EventuallyEq.refl _ fun _ => 0 }
-
-instance divisionSemiring [DivisionSemiring β] : DivisionSemiring β* where
-  toSemiring := Germ.semiring
-  __ := Germ.groupWithZero
-
-instance divisionRing [DivisionRing β] : DivisionRing β* :=
-  { Germ.ring, Germ.divisionSemiring with qsmul := qsmulRec _ }
-
-instance semifield [Semifield β] : Semifield β* :=
-  { Germ.commSemiring, Germ.divisionSemiring with }
-
-instance field [Field β] : Field β* :=
-  { Germ.commRing, Germ.divisionRing with }
+instance instGroupWithZero [GroupWithZero β] : GroupWithZero β* where
```

Comment:
Some of these are probably in the wrong order but out of scope if reverted.

Reply:
I don't understand what you mean

Reply:
I don't think this change is in scope for a PR that renames this much stuff...

Reply:
@YaelDilles: @mattrobball means that either he wants this changed in the way he states, or you should just revert all the changes here because they don't belong in this PR in the first place.